source/operations/install-deploy-manage/migrate-fs-gateway.rst

```diff
@@ -0,0 +1,203 @@
+      :class: copyable
+
+      mc admin policy list ALIAS --json
+
+   - Replace ``ALIAS`` with the alias for the existing standalone deployment.
+
+   Use the list to recreate the policies on the new deployment.
+
+#. Duplicate **groups** from existing standalone deployment to new deployment
+
+   Use :mc-cmd:`mc admin group list` with the ``--json`` flag to retrieve a list of groups that exist on the standalone deployment.
+
+   .. code-block:: shell
+      :class: copyable
+
+      mc admin group list ALIAS --json
+
+   - Replace ``ALIAS`` with the alias for the existing standalone deployment.
+
+   Use the list to recreate the groups on the new deployment.
+
+#. Duplicate **users** from existing standalone deployment to new deployment
+
+   Use :mc-cmd:`mc admin user list` with the ``--json`` flag to retrieve a list of users with access key, policy name, and status.
+
+   .. code-block:: shell
+      :class: copyable
+
+      mc admin user list ALIAS --json
+
+   - Replace ``ALIAS`` with the alias for the existing standalone deployment.
+
+   Use the list to recreate the users on the new deployment.
+
+   Note: You will need to define each user's ``SECRETKEY`` on the new deployment.
+   Make note of the ``SECRETKEY`` for each user to let them know their new credentials.
+
+#. Duplicate **service accounts** from existing standalone deployment to new deployment
+
+   Use :mc-cmd:`mc admin user svcacct list` with the ``--json`` flag to list existing service accounts on the standalone deployment.
+
+   .. code-block:: shell
+      :class: copyable
+
+      mc admin user svcacct list ALIAS --json
+
+   - Replace ``ALIAS`` with the alias for the existing standalone deployment.
+
+   Use the list to recreate the service accounts on the new deployment.
+
+#. Use :mc:`mc mirror` with the :mc-cmd:`~mc mirror --preserve` and :mc-cmd:`~mc mirror --watch` flags on the standalone deployment to move objects to the new |SNSD| deployment
+
+   .. code-block:: shell
+      :class: copyable
+
+      mc mirror --preserve --watch SOURCE/BUCKET TARGET/BUCKET
+
+   - Replace ``SOURCE/BUCKET`` with the alias and a bucket for the existing standalone deployment.
+   - Replace ``TARGET`` with the alias and corresponding bucket for the new deployment.
+
+#. After the initial mirror process completes, convert standalone deployment to be read only
+
+   One options for doing this is to remove policies that provide write permissions from users or groups with :mc:`mc admin policy`'s ``unset`` command.
+
+   .. code-block:: shell
+
+      mc admin policy unset ALIAS POLICYNAME [user=USERNAME | group=GROUPNAME]
+
+   - Replace ``ALIAS`` with the alias for the existing standalone deployment 
+   - Replace ``POLICYNAME`` with the name of a policy on the existing standalone deployment.
+   - Replace either ``USERNAME`` or ``GROUPNAME`` with either the user or group assigned the policy.
+
+   Use care to not remove permissions for the user running the ``mc mirror`` command.
```

Comment:
IMO this is too risky. It's enough to ask users to stop clients from writing to this deployment any further.

Reply:
Cut.